Sources/WebSocket.swift

```diff
@@ -263,8 +263,10 @@ open class FoundationStream : NSObject, WSStream, StreamDelegate  {
     #if os(Linux) || os(watchOS)
     #else
     public func sslTrust() -> (trust: SecTrust?, domain: String?) {
-        let trust = outputStream!.property(forKey: kCFStreamPropertySSLPeerTrust as Stream.PropertyKey) as! SecTrust?
-        var domain = outputStream!.property(forKey: kCFStreamSSLPeerName as Stream.PropertyKey) as? String
+        guard let outputStream = outputStream else { return (nil, nil) }
+
+        let trust = outputStream.property(forKey: kCFStreamPropertySSLPeerTrust as Stream.PropertyKey) as! SecTrust?
+        var domain = outputStream.property(forKey: kCFStreamSSLPeerName as Stream.PropertyKey) as! String?
```

Comment:
we usually follow the pattern `property as? String`, instead of the `as!` in the middle. I believe it gives the same result.

Reply:
You're suggesting the existing code... https://github.com/adamkaplan/Starscream/blob/master/Sources/WebSocket.swift#L267

I understand optional casts as a general best practice, however it isn't practical to use `as?` here for the reasons I explained in detail in the description above and the linked issue.

(it crashes! To now crash it would be`as? String?`, but then you end up with a `String??`...